utils/pipeline_utils.py

```diff
@@ -59,36 +47,56 @@ def load_pipeline(args):
     if args.compile_unet:
         pipe.unet.to(memory_format=torch.channels_last)
         print("Compile UNet")
-
+        swap_conv2d_1x1_to_linear(pipe.unet)
```

Comment:
How much does it help? Should we make it configurable?

Reply:
not a lot to be honest, the shapes looked like they would be good to apply quantization to but it seems the improvements are relatively minor. Note i've altered the api's so they can be called on individual modules if needed so you could call that on individual conv modules we'd like to target.

Reply:
also,the code has torch._inductor.config.conv_1x1_as_mm = True so its already doing this, but just at a point after we apply quantization. Its unclear if the two different ways of doing conv1x1->linear have different perf on a shape to shape basis but overall we can see that the perf impact is relatively minor from [perf results](https://github.com/huggingface/sdxl-fast/pull/2#issuecomment-1855383345)